src/App.tsx

```diff
@@ -1,27 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const allGoods = () => {
+    getAll().then(value => setGoods(value));
+  };
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  const fiveFirst = () => {
+    get5First().then(value => setGoods(value));
+  };
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+  const allRed = () => {
+    getRedGoods().then(value => setGoods(value));
+  };
```

Comment:
use correct naming for event handlers
<img width="281" alt="Screenshot 2022-07-26 at 09 40 32" src="https://user-images.githubusercontent.com/99331793/180940554-f42ef656-b8ec-48a9-80e9-dfce34b1f33a.png">
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364